contrib/lib/tests/space_helmet.rs

```diff
@@ -0,0 +1,97 @@
+#![cfg_attr(test, feature(plugin, decl_macro))]
+#![cfg_attr(test, plugin(rocket_codegen))]
+#![feature(extern_prelude)]
+
+extern crate rocket_contrib;
+extern crate rocket;
+
+#[cfg(feature = "space_helmet")]
+mod space_helmet_tests {
+    use rocket_contrib::space_helmet::*;
+    use rocket::http::uri::Uri;
+    use rocket::http::Status;
+    use rocket::local::Client;
+
+    #[get("/")]
+    fn hello() -> &'static str {
+        "Hello, world!"
+    }
+
+    macro_rules! check_header {
+        ($response:ident, $header_name:expr, $header_param:expr) => {
+            match $response.headers().get_one($header_name) {
+                Some(str) => {
```

Comment:
Don't name the variable `str`, since that's a type name. I like using `string`. Also, there's no need for the `{ }` block here.